packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambdaLayerWalkthrough.ts

```diff
@@ -112,6 +116,8 @@ export async function updateLayerWalkthrough(
     }
   }
   if (islayerVersionChanged) {
+    context.print.info('');
+    context.print.warning('Please update your Lambda function depending on this layer using - amplify function update');
```

Comment:
Maybe adding quotes would be better? cc @brene 
```js
context.print.warning('Please update your Lambda function depending on this layer using - "amplify function update"');
```

Reply:
New Lambda layer version created. Any function that wants to use the latest layer version need to configure it by running - "amplify function update"